public/css/breach-detail.css

```diff
@@ -227,8 +227,7 @@ button.fade-out {
   .rec-ip-non-us,
   .rec-ip-us,
   .rec-sec-qa,
-  .rec-dob,
-  .rec-pins {
+  .rec-dob {
```

Comment:
Almost there! Looks like we just need to remove the `.rec-dob` selector here and then we'll be all set.